### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by PitchBlecc** [genekogan/KinectProjectorToolkit#8]
After launching I get several NullPointerExceptions.
I'm using:
 Processing 2.2.1
 SimpleOpenNI Version 1.96
 OpenCV for Processing 0.5.2
 Using Java OpenCV 2.4.5.0
 ControlP5 2.2.5

The Simple Open NI Demos run without any errors. Do you have an idea on how to resolve the Exceptions?

**Comment by genekogan** [comment]
can you provide some more details, like where you are getting the null pointers?

**Comment by johnnyshankman** [comment]
Same issue

**Comment by johnnyshankman** [comment]
I don't get a line number just a few logs of java.lang.NullPointerException

**Comment by johnnyshankman** [comment]
@genekogan what was the last known working versions?

**Comment by johnnyshankman** [comment]
Alright I uninstalled Processing, reinstalled the most stable 2.x version from their site, reinstalled all the libraries and now everything seems to be hunky dory.

I still get the errors but everything seems to be working fine.

**Comment by genekogan** [comment]
yes, KPT was never really updated for Processing 3, so in 2.2 at least everything should run.
